test/unit/spec/LoginRouter_spec.js

```diff
@@ -1141,14 +1140,19 @@ function (Okta, Q, Logger, Errors, BrowserFeatures, WidgetUtil,
                 error_description: 'Invalid value for client_id parameter.'
               }
             });
-            return tick();
+            return tick(test);
           })
-          .then(function () {
-            expect(errorSpy.calls.count()).toBe(1);
-            var err = errorSpy.calls.argsFor(0)[0];
-            expect(err instanceof Errors.OAuthError).toBe(true);
-            expect(err.name).toBe('OAUTH_ERROR');
-            expect(err.message).toBe('Invalid value for client_id parameter.');
+          .then(function (test) {
+            expect(test.afterErrorHandler).toHaveBeenCalledTimes(1);
+            expect(test.afterErrorHandler.calls.allArgs()[0]).toEqual([
+              {
+                controller: 'mfa-verify'
```

Comment:
curious why it's `mfa-verify`

Reply:
Because `setupOAuth2` goes use mfa-verify https://github.com/okta/okta-signin-widget/blob/54bee338d94c35bc3709f69c403090cf645e7ee6/test/unit/spec/LoginRouter_spec.js#L132